index.js

```diff
@@ -1,131 +1,18 @@
-  console.log(`   _________________________________________________________
-  //=======================================================\\
-  \\..............WELCOME TO THE ONE AND ONLY..............//
-  //.................---------------------.................\\
-  \\.................||  GUESS WHAT!!!  ||.................//
-  //.................---------------------.................\\
-  \\=======================================================//
-   <><><><><><><><><><><><><><>*<><><><><><><><><><><><><><>\n\n`);
+start();
 
+async function start() {
   console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.")
-  await numInt();
-  async function numInt() {
-    let secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
-    if (isNaN(parseInt(secretNumber)) || parseFloat(secretNumber) % 1 != 0) {    //Ensures user enters an integer
-      console.log("Please enter an integer...\n");
-      await numInt();
-    } else {
-      console.log('You entered: ' + secretNumber);
-      await playComp(secretNumber);
-    }
-  }
-}
-
-function guess(min, max) {
-  return Math.floor((parseInt(max)+min)/2); //Every guess is the middle of the remaining range of numbers
-}
-
-async function correctLetter (string, option1, option2) {
-  if (string.toUpperCase() === option1 || string.toUpperCase() === option2) {
-    return string.toUpperCase();
-  } else {
-    while (string.toUpperCase() !== option1 || string.toUpperCase() !== option2) {
-      string = await ask('Please enter a correct option...\n');
-      if (string.toUpperCase() === option1 || string.toUpperCase() === option2) {
-        return string.toUpperCase();
-      }
-      }
-  }
-}
-
-async function playComp(num) {
-  let numGuess = min + Math.floor(Math.random()*(max-min+1)); //First guess should be random to not give away our secret!
-  determine();
-  async function determine() {
-      let yesNo = await ask(`Is the number .... ${numGuess}? Y or N?\n`);
-      //if (yesNo.toUpperCase() === 'Y') {
-      if (await correctLetter(yesNo, 'Y', 'N') === 'Y') {
-        guessCount += 1;
-        console.log(`Got it! Your number is ${numGuess}! \nIt took me ${guessCount} guesses!\n`);
-        console.log("Let's play another game where I (computer) think of a number and you (human) try to geuss it.\n");
-        max = process.argv[2];
-        min = 1;
-        guessCount = 0;
-        await playHuman();
-      //} else if (yesNo.toUpperCase() === 'N') {
-      } else if (await correctLetter(yesNo, 'Y', 'N') === 'N') {
-        if (numGuess === parseInt(num)) {
-          console.log(`You think you are tricky don't you, but I know that your number is ${numGuess}!`); //determines if player lies about answer
-          guessCount +=1;
-          console.log(`It took me ${guessCount} guesses!\n`)
-          max = process.argv[2];
-          min = 1;
-          guessCount = 0;
-          await playHuman();
-        } else {
```

Comment:
Make this an `else if` instead of just an `else` and you can then use an `else` later to handle unexpected input much more simply than you currently are.